Source/SpeechToTextV1/SpeechToText+Recognize.swift

```diff
@@ -109,8 +106,14 @@ extension SpeechToText {
         session.defaultHeaders = defaultHeaders
 
         // set callbacks
-        session.onResults = success
-        session.onError = failure
+        session.onResults = { result in
+            var response = WatsonResponse<SpeechRecognitionResults>(statusCode: 0)
```

Comment:
I expect a lot of users would be confused to see a status code of `0`. Why did you choose this instead of `200`?

Reply:
Agreed.  I will fix.